index.js

```diff
@@ -68,7 +68,9 @@ const D2LHypermediaRels = {
 		checklist: 'https://checklists.api.brightspace.com/rels/checklist',
 		checklistItem: 'https://checklists.api.brightspace.com/rels/checklist-item'
 	},
-	Content: {},
+	Content: {
+		sequence: 'https://api.brightspace.com/rels/sequence'
```

Comment:
As @CodeBaboon mentioned, this should be under the parent object.

For some history/context, we started using rels in the HM APIs before this library existed, and we were initially just creating everything under the top-level `api.brightspace.com/rels` namespace. At some point, we got all "Zen of Python"-ed up and started namespacing them (e.g. `content.api.brightspace.com/rels`), but a lot of the older ones were never moved into their more correct namespaces (probably not worth doing, as it would be a breaking change to remove the old rel)